### DISCUSSION TITLE
Do you have a GIT.IGNORE file for Skip

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#208] [Q&A] [answered]
I'm not sure which directories I should NOT check into a GIT Repo.

Do you have recommendations for git.ignore file?

**Comment by marcprux** [comment, marked as answer]
I recommend using the same one we use for our projects: https://github.com/skiptools/skipapp-showcase/blob/main/.gitignore

**Reply by ptrkstr** [reply]
@marcprux could I suggest that the gitignore is either added by default during project init or at least asked? There are a lot of files generated using Skip and it's only a matter of time before people start experiencing problems by ignoring the wrong files and start raising issues here.
Maybe the recommended gitignore could also be added to the docs.

**Reply by marcprux** [reply]
Good idea! We will have `skip init` create a `.gitignore` file in the next release. Thanks for the suggestion!
